frontend/src/metabase/dashboard/reducers.unit.spec.js

```diff
@@ -31,6 +31,7 @@ describe("dashboard reducers", () => {
         loadingStatus: "idle",
       },
       parameterValues: {},
+      draftParameterValues: null,
```

Comment:
I opt for `null` rather than an empty object so that in the view I can simply use this `draftParameterValues ?? parameterValues`

Reply:
Feels weird to have different behavior for `parameterValues` and draft ones, I'd use `{}` and then `_.isEmpty(draftParameterValues) ? parameterValues : draftParameterValues`